src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -496,7 +496,7 @@ public function get($id, $invalidBehavior = ContainerInterface::EXCEPTION_ON_INV
      * $container = new ContainerBuilder(array('foo' => 'bar'));
```

Comment:
this one seems wrong as well https://github.com/symfony/symfony/blob/75d5cb1bad123378a0b2202647dd172609a7a541/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L94